src/hotspot/share/prims/whitebox.cpp

```diff
@@ -2728,6 +2729,14 @@ WB_ENTRY(jlong, WB_Rss(JNIEnv* env, jobject o))
   return os::rss();
 WB_END
 
+WB_ENTRY(void, WB_ControlledCrash(JNIEnv* env, jobject o, jint how))
+#ifdef ASSERT
+  DEBUG_ONLY(VMError::controlled_crash(how);)
```

Comment:
`DEBUG_ONLY` wrapping isn't needed here, since we're in `#ifdef ASSERT` context.

Reply:
Thanks, that was a remnant from an earlier version. Fixed.